samples/tooling/profiles/profiles.cpp

```diff
@@ -145,6 +150,55 @@ std::unique_ptr<vkb::Instance> Profiles::create_instance(bool headless)
 	}
 
 	VkInstanceCreateInfo create_info{};
+
+#if (defined(VKB_ENABLE_PORTABILITY))
+	uint32_t instance_extension_count;
+	VK_CHECK(vkEnumerateInstanceExtensionProperties(nullptr, &instance_extension_count, nullptr));
+	std::vector<VkExtensionProperties> available_instance_extensions(instance_extension_count);
+	VK_CHECK(vkEnumerateInstanceExtensionProperties(nullptr, &instance_extension_count, available_instance_extensions.data()));
+
+	// If VK_KHR_portability_enumeration is available at runtime, enable the extension and flag for instance creation
+	if (std::any_of(available_instance_extensions.begin(),
+	                available_instance_extensions.end(),
+	                [](VkExtensionProperties extension) { return strcmp(extension.extensionName, VK_KHR_PORTABILITY_ENUMERATION_EXTENSION_NAME) == 0; }))
```

Comment:
Maybe you should use `VkExtensionProperties const & extension` here, to prevent copying the structs.

Reply:
Thanks for the good suggestion.  Fixed in https://github.com/KhronosGroup/Vulkan-Samples/pull/1119/commits/e6d176123fcb4e4f3ea9f6d93c928cb5841f1929.